cve_bin_tool/cli.py

```diff
@@ -205,6 +205,19 @@ def main(argv=None):
         help="provide sbom filename",
         default="",
     )
+    input_group.add_argument(
+        "--vex",
+        action="store",
+        choices=["cyclonedx", "csaf", "open"],
```

Comment:
@anthonyharrison I think its better this way so that lib4vex don't have to figure out on auto and auto being handy if no value for --vex is passed, should i remove it so, for parser to always work on auto?

Reply:
If auto is accurate enough (and I'm fairly sure it is?) we should probably use auto and get rid of the option.   It'll take a milisecond for the computer to figure it out and a lot longer than that for the human if they don't already know the answer. ;)

Reply:
@terriko i have shifted the manual defining of vex to auto   ; )